test/services/WhatDoYouWantToDoServiceSpec.scala

```diff
@@ -16,19 +16,19 @@
 
 package services
 
-import base.SpecBase
-import connectors._
+import base.{AppWithDefaultMockFixtures, SpecBase}
+import connectors.*
 import generators.Generators
 import models.{Availability, Feature}
 import org.mockito.ArgumentMatchers.any
 import org.mockito.Mockito.{reset, verify, when}
 import org.scalacheck.Arbitrary.arbitrary
-import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks.forAll
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-class WhatDoYouWantToDoServiceSpec extends SpecBase with ScalaCheckPropertyChecks with Generators {
+class WhatDoYouWantToDoServiceSpec extends SpecBase with AppWithDefaultMockFixtures with Generators {
```

Comment:
Does this need AppWithDefaultMockFixtures? Could we get around using app.injector.instanceOf ... ?